thoth/common/openshift.py

```diff
@@ -469,6 +469,14 @@ def get_configmaps(self, namespace: str, label_selector: str) -> Dict[str, Any]:
         )
         return result
 
+    def get_image_streams(self, namespace: str, label_selector: str) -> Dict[str, Any]:
+        """Get all image streams in a namespace and select them by label."""
+        v1_image_streams = self.ocp_client.resources.get(api_version="v1", kind="ImageStream")
+        result: Dict[str, Any] = v1_image_streams.get(
+            label_selector=label_selector, namespace=namespace
+        )
+        return result
```

Comment:
Will it retrieve a list? (IIRC there is an "items" key?)

Reply:
No a Dict[str, Any] as for configmaps.